confluent_kafka/__init__.py

```diff
@@ -634,3 +634,26 @@ def __repr__(self):
 
     def __str__(self):
         return "{}".format(self.partition)
+
+
+class ThrottleEvent (object):
+    """
+    ThrottleEvent passed as an argument to throttle_cb.
+    Contains details about requests throttled by the broker.
+
+    This class is typically not user instantiated.
+
+    :ivar broker_name: The hostname of the broker which throttled the request
+    :ivar broker_id: The broker's id
+    :ivar throttle_time_ms: The delay associated with the request(seconds)
```

Comment:
"The amount of time the broker throttled (delayed) the request before responding (seconds)"